CONTRIBUTING.md

```diff
@@ -51,3 +52,49 @@ A couple of notes:
 
 - Using a naming convention will allow us to verify that APIs marked `testsOnly` are not being used inside the SDK; we’ll do this in #70.
 - I believe that we should be able to eliminate the boilerplate of re-exposing a `private` member as a `testsOnly` member (as exemplified above) using a macro (called e.g. `@ExposedToTests`), but my level of experience with macros is insufficient to be confident about being able to do this quickly, so have deferred it to #71.
+
+#### Attributing tests to a spec point
+
+When writing a test that relates to a spec point from the Chat SDK features spec, add a comment that contains one of the following tags:
+
+- `@spec <spec-item-id>` — The test case directly tests all the functionality documented in the spec item.
+- `@specOneOf(m/n) <spec-item-id>` — The test case is the m<sup>th</sup> of n test cases which, together, test all the functionality documented in the spec item.
```

Comment:
**suggestion (documentation):** Replace Markdown superscript syntax with plain text

The superscript syntax '<sup>th</sup>' won't render correctly in plain text. Consider using 'mth' instead.